cmd/rbac-manager/main.go

```diff
@@ -27,7 +27,7 @@ var (
 	app     = kingpin.New("rbac-manager", "Manages rbac.crd.gocardless.com resources").Version(Version)
 	refresh = app.Flag("refresh", "Refresh interval checking directory sources").Default("1m").Duration()
 
-	commonOpts = cmd.NewCommonOptions(app)
+	commonOpts = cmd.NewCommonOptions(app, true)
```

Comment:
Is it an anti-pattern to do this? 

```
exposeMetrics := true
commonOpts = cmd.NewCommonOptions(app, exposeMetrics)
```

Reply:
You can make this more explicit if you want. I'll help you refactor in a bit